app/helpers/games_helper.rb

```diff
@@ -32,4 +32,12 @@ def render_piece(x, y)
       piece.white? ? white_pawn : black_pawn
     end
   end
+
+  def player(color)
+    if color == 'white'
+      User.find(current_game.player_white_id)
```

Comment:
alternate way to write this, would be `current_game.player_white`

Which does the same thing, but uses Rails built in accessor for player_white (which is created since we have an association between the player and game)